package.json

```diff
@@ -61,5 +64,8 @@
     "All": {
       "description": "Test with any doctype."
     }
+  },
+  "dependencies": {
+    "pouchdb": "6.1.1"
```

Comment:
Maybe it has to be in `devDependencies` if tests used it. Or in `peerDependencies` if not. I am not a pro with that.